build_image.sh

```diff
@@ -15,14 +15,14 @@ fi
 setup_type=$1
 app_path=$PWD
 app_name=${app_path##*/}
-parent_path=`dirname $PWD`
+#parent_path=`dirname $PWD`
```

Comment:
Let's get rid of this line if it's no longer needed

Reply:
Acknowledged!